README.md

```diff
@@ -16,12 +16,16 @@ package main
 
 import (
 	"fmt"
+	"log"
+
 	"github.com/VojtechVitek/go-trello"
 )
 
 func main() {
 	// New Trello Client
-	trello, err := trello.NewAuthClient("application-key", "token")
+	appKey := "application-key"
+	token := "token"
+	trello, err := trello.NewAuthClient(appKey, &token)
```

Comment:
Note: I'd like to change the API to accept token string instead of the pointer and have `trello.UpdateToken()` method (as discussed in the previous PRs with @eparis).